svc-task/tresponse/task.go

```diff
@@ -43,7 +44,7 @@ type Task struct {
 	StartTime       time.Time   `json:"StartTime"`
 	EndTime         time.Time   `json:"EndTime,omitempty"`
 	TaskStatus      string      `json:"TaskStatus"`
-	SubTasks        string      `json:"SubTasks,omitempty"`
+	SubTasks        ListMember  `json:"SubTasks,omitempty"`
```

Comment:
since it's a structure change it to the pointer so that if ti's nil omitempty will be applicable

Reply:
Done